src/week1/day0/form.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html lang="en">
+   <head>
+      <meta charset="utf-8">
+      <title> Form </title>
+      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+   </head>
+
+   <body>
+      <header>
+         <h1> Form </h1>
+      </header>
+      <section>
+        <h2> This is sign up form </h2>
+         <form class="signup-form">
+            First name:
+            <input type="text" name="firstname" value=""><br>
```

Comment:
- don't use BR
- try to use label tag as a wrapper
- don't forget to link the label with the input (focus)